var/spack/repos/builtin/packages/sim-model/package.py

```diff
@@ -97,21 +97,24 @@ def _build_mods(self, mods_location, link_flag='', include_flag='',
                                mods_location)
 
         assert os.path.isfile(os.path.join(output_dir, 'special'))
-        return include_flag, link_flag
+        return include_flag_raw, link_flag_raw
+
+    def _nrnivmodlcore_params(self, include_flag, link_flag):
+        return ['-n', self.mech_name, '-i', include_flag, '-l', link_flag, '-V']
+
+    def _add_corenrn_include_flag(self):
```

Comment:
```suggestion
    def _add_coreneuron_include_flag(self):
```
There's also `corenrn` elsewhere. Maybe we could refer to CoreNEURON without any abbreviation for clarity?